.jenkins/pytorch/test.sh

```diff
@@ -181,10 +181,24 @@ test_xla() {
   assert_git_not_dirty
 }
 
+test_backward_compatibility() {
```

Comment:
It would probably be a good idea to say in big blinky letters that this uninstalls your torch install, so you had better not try to run any other tests afterwards. Another way to make this safer is to uninstall the torch/torchvision nightly after you're done, so we don't accidentally pick up the wrong torch version later.

Reply:
yes, added comments